api/proto/controller/v1/controller_service.proto

```diff
@@ -0,0 +1,36 @@
+syntax = "proto3";
+package controller.v1;
+
+import "common/v2/common_v2.proto";
+
+option go_package = "github.com/Layr-Labs/eigenda/api/grpc/controller/v1";
+
+// ControllerService defines the APIs for the controller.
+//
+// Currently, this API is only intended for *internal* consumption: this is a way for different parts of the disperser
+// to communicate with each other
+service ControllerService {
+  // AuthorizePayment handles payment authorization for blob dispersal
+  //
+  // This is intended to be called by API server instances that are handling dispersal requests. The controller
+  // is responsible for accounting and metering for the dispersal.
+  rpc AuthorizePayment(AuthorizePaymentRequest) returns (AuthorizePaymentResponse) {}
+}
+
+// Contains all information necessary for the controller to evaluate the validity of a dispersal payment
+message AuthorizePaymentRequest {
+  // The blob header is used for the following purposes:
+  // 1. Contains the PaymentHeader, which describes the payment being offered
+  // 2. Contains the quorums being dispersed to
+  common.v2.BlobHeader blob_header = 1;
+
+  // Signature over the keccak hash of this AuthorizePaymentRequest (excluding this signature field).
+  bytes disperser_signature = 2;
```

Comment:
We should include a timestamp in any protobuf where we include a signature. This is needed by the replay gaurdian to prevent signature replay attacks. 

This is likely getting into overkill territory from a security point of view. But perhaps overkill is justified for gRPCs, since gRPCs are one of the most likely points of attack in a system.

Reply:
For the type of the timestap, I suggest using a `uint64` holding the time in seconds since the epoch.